Automattic/jetpack
# Docs: Updating monorepo.md with extra mirror repo information

#### Changes proposed in this Pull Request:

* Updating monorepo.md with a few additional bits of information relevant to setting up a mirror repo.

#### Other information:


#### Jetpack product discussion

Information taken from here: p1657635173682769-slack-C01U2KGS2PQ

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Proof-read - and can also check the linked Slack thread to make sure the information is correct.
